antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -1050,3 +1051,35 @@ def initialize_additional_data(self, variant_study: VariantStudy) -> bool:
                 exc_info=e,
             )
             return False
+
+    def clear_all_snapshots(self, limit: int, params: t.Optional[RequestParameters] = None) -> None:
```

Comment:
For `Mypy` you should deal first with the case `params=None` (it is an Optional parameter)